### PR TITLE
Print car ID only labels using 1/2" round dots

### PR DESCRIPTION
**Pull request by ajspeck**
This pull request adds two features:

1) It adds the option to print small car id only labels on Garagesalepup AWE labels ( https://www.garagesalepup.com/templates/we_templates.htm ).  Note I was unable to get the auto-layout functionality to work for the spacing on these labels and thus added a function to set the layout manually.
2) It sorts the printable labels in the same order as chosen on the selection page.

### CODE REVIEW

File: website/print/render/racer.inc [hunk 1]
```diff
@@ -97,7 +102,11 @@ if (!is_subclass_of(document_class(), 'PrintableRacerDocument')) {
       if (isset($_GET['where'])) {
         $sql = $sql.' WHERE '.$_GET['where'];
       }
-      $sql = $sql.' ORDER BY lastname, firstname, carnumber';
```
Comment (jeffpiazza) [inline website/print/render/racer.inc]: This whole 'else' clause is something of an unfinished thought.  When invoked from print.php, $_GET['ids'] is always set, so we never get here anyway.  I'm not super-enthusiastic about building it out unless you have a use case I'm not seeing.

File: website/js/print.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ function print_selected() {
   
   window.open("render-document.php/" + doc_class_details['type'] + "/" + doc_class
               + "?options=" + encodeURIComponent(JSON.stringify(options))
+              + "&sort=" + encodeURIComponent($("#sortorder-racers option:selected").val())
```
Comment (jeffpiazza) [inline website/js/print.js]: See below.  Ordering is determined entirely by the order in which the ids appear.

File: website/print/docs/racer/car-label-AWE/document.inc [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<?php
+// Print 1/2 circular labels, compatible with GarageSalePup AWE.
+class CarLabelAWE extends PrintableRacerDocument {
+  function name() { return "Car Label GarageSalePup AWE"; }
```
Comment (jeffpiazza) [inline website/print/docs/racer/car-label-AWE/document.inc]: Could we say something like "Car number 1/2-inch round (GarageSalePup.com)" here instead?

File: website/print/render/racer.inc (hunk 1 above)
Comment (ajspeck) [reply website/print/render/racer.inc]: Sorry I didn't realize that the ids now always are set -- I agree that this is a better solution so I have reverted these changes.

File: website/js/print.js (hunk 2 above)
Comment (ajspeck) [reply website/js/print.js]: See previous -- I reverted this change.

File: website/print/docs/racer/car-label-AWE/document.inc (hunk 3 above)
Comment (ajspeck) [reply website/print/docs/racer/car-label-AWE/document.inc]: I changed this to "Car Number Label (GarageSalePup 1/2 inch round)"

**Comment by jeffpiazza** [comment]
Thanks for this

**Pull request merged**